libkbfs/keybase_service_base.go

```diff
@@ -437,19 +445,68 @@ func (k *KeybaseServiceBase) LoadUserPlusKeys(ctx context.Context,
 	return k.processUserPlusKeys(res)
 }
 
+var allowedLoadTeamRoles = map[keybase1.TeamRole]bool{
+	keybase1.TeamRole_NONE:   true,
```

Comment:
NONE is used when the caller doesn't care what role the user has.  Right now it's treated the same as READER, but that might not be the case in the future.  It's also used as a zero value when the desiredUID is unset.